<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бінарні дерева</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="pushEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Додати в дерево</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delButton">
        <property name="text">
         <string>Видалити</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="seeButton">
        <property name="text">
         <string>Шукати</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pointButton">
        <property name="text">
         <string>Визначити</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дії</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Завдання</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_3"/>
   <addaction name="action_2"/>
   <addaction name="action_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Оновити</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Видалення елемента</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Пошук елемента</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Кількість входжкнь</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Обчислити добуток всіх елементів</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushEdit</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
